kahuna/app/lib/KahunaConfig.scala

```diff
@@ -39,6 +39,15 @@ class KahunaConfig(resources: GridConfigResources) extends CommonConfig(resource
   val restrictDownload: Option[Boolean] = booleanOpt("restrictDownload")
   val useReaper: Option[Boolean] = booleanOpt("useReaper")
 
+  // interim permissions filter configuration settings
+  val usePermissionsFilter: Option[Boolean] = booleanOpt("usePermissionsFilter")
+  val usageRightsSummary: Option[Boolean] = booleanOpt("usageRightsSummary")
+  val permissionsOptions: Option[String] = stringOpt("permissionsOptions").filterNot(_.isEmpty)
+  val permissionsLabels: Option[String] = stringOpt("permissionsLabels").filterNot(_.isEmpty)
+  val permissionsMappings: Option[String] = stringOpt("permissionsMappings").filterNot(_.isEmpty)
```

Comment:
I haven't looked over this PR at all, but @paperboyo and I ended up discussing the config aspect when he was playing with this PR in our TEST environment... I was wondering why we're not using HOCON arrays here (https://docs.tibco.com/pub/str/latest/doc/html/hocon/hocon-syntax-reference.html) but instead splitting on `,` & `#` AFAICT (which is harder to read/configure but also means command and hash cannot be used in the search term either)